.github/workflows/_win-build.yml

```diff
@@ -43,7 +48,7 @@ jobs:
   build:
     # Don't run on forked repos.
     if: github.repository_owner == 'pytorch'
-    runs-on: [self-hosted, windows.4xlarge.nonephemeral]
+    runs-on: ${{ inputs.runner }}
```

Comment:
windows runners don't work on LF accounts yet. We need to get custom Windows AMIs working there first